CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.20.0] - 2022-03-04
+### Added
+- Added substring matching for targeting. See [README.md](README.md#Matching-Values) for usage examples.
```

Comment:
I changed the PR title to "Add Regexp Targeting". Can you change it here in the CHANGELOG and anywhere else in the PR?

Reply:
Fixed [here](https://github.com/Invoca/process_settings/pull/115/commits/c2e5de20970435ffe3ac81416505ef8c7f855ff7)